backend/LexBoxApi/Auth/Attributes/LexboxAuthAttribute.cs

```diff
@@ -9,10 +10,17 @@ public abstract class LexboxAuthAttribute : DescriptorAttribute, IAuthorizeData
 {
     public LexboxAuthAttribute(string policy)
     {
-        Policy = policy;
+        _policy = policy;
     }
 
-    public string Policy { get; set; }
+    private string _policy;
+
+    [AllowNull]
```

Comment:
I don't think this is right. `AllowNull` [should mean](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/attributes/nullable-analysis#preconditions-allownull-and-disallownull) that it's ok if someone sets this to null even though the property is non-nullable, because we default to a non-null value. But we don't allow `null`s.

And the constructor parameter is also non-nullable, so why wouldn't you assign it to the property and make use of the null-check there?

```suggestion
```

Reply:
Yeah this is ugly, but it's because the IAuthorizeData interface allows setting the Policy to null, so this is required otherwise we get a warning saying that null can be set here due to the interface even though the property is non nullable.